README.md

```diff
@@ -45,7 +45,7 @@ Quinn was created and is maintained by Dirkjan Ochtman and Benjamin Saunders.
 
 ## Status
 
-- [x] QUIC draft 27 with TLS 1.3
+- [x] QUIC draft 31 with TLS 1.3
```

Comment:
Another commit mentions draft 32, but you used 31 first. Which is it? Are they different (enough)?

Reply:
2bff792 is from after draft 31 was cut, but draft 32 has not been formally released yet.

Reply:
I have email from the mailing list announcing draft 32, from Oct 21st. How do you determine it hasn't been released?

Reply:
Whoops, good catch.